tests/test_parsers.py

```diff
@@ -410,3 +410,26 @@ def test_round_trip(self):
 
         # This checks that nothing funny gets added unexpectedly
         self.assertEqual(expected_prefix_map, reconsitited_msdf.prefix_map)
+
+    def test_round_trip_tsv_json_tsv(self):
+        """Test TSV => JSON => TSV using convert() + parse()."""
+        input_tsv = test_data_dir.joinpath("basic_subset.tsv")
+        input_msdf = parse_sssom_table(input_tsv)
+        input_msdf.clean_prefix_map()
+        with TemporaryDirectory() as directory:
+            directory = Path(directory)
+            json_path = directory.joinpath("basic_subset.json")
+            tsv_path = directory.joinpath("json_to_tsv.tsv")
+            with json_path.open("w") as jfile:
+                convert_file(input_path=input_tsv, output_format="json", output=jfile)
+            with tsv_path.open("w") as tfile:
+                parse_file(
```

Comment:
Why use the wrapper method rather than parse_sssom_json?

Reply:
I dont see a big problem with this, but it seems like it is adding a variable into this test (which is "the mechanism by which `parse_file` determines which parsing function to call"), which should ideally be tested in a another specialised test, but I guess you could argue that this test "more closely approximates the behaviour of CLI". So I will leave it at that.

Reply:
Exactly.